packages/core/src/header-builder/header-builder.ts

```diff
@@ -1,15 +1,16 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
 import { MapType } from '@sap-cloud-sdk/util';
-import { Destination } from '../../scp-cf';
-import { ODataRequestConfig, ODataRequest, isWithETag } from '../request';
+import { Destination } from '../scp-cf';
+import { ODataRequestConfig, isWithETag } from '../odata/common/request';
+import { ODataRequest } from '../odata/common/request/odata-request';
+import { buildAuthorizationHeaders } from './authorization-header';
 import { buildCsrfHeaders } from './csrf-token-header';
 import {
-  filterNullishValues,
+  getHeader,
   replaceDuplicateKeys,
-  getHeader
+  filterNullishValues
```

Comment:
minor: use linter to sort imports alphabetically.

Reply:
I did, our linter currently does not do that. ;)